bin/transform

```diff
@@ -293,6 +355,12 @@ if __name__ == '__main__':
     gisaid_data = parse_sex(gisaid_data)
     gisaid_data = update_metadata(gisaid_data)
 
+    # Warn about future dates
+    try:
+        warn_future_dates(gisaid_data)
+    except HTTPError:
+        LOG.info("Slack message couldn't be sent, but continuing anyway.")
```

Comment:
The whole `warn_future_dates` function feels like it should be a separate script that runs after this one. Particularly since it doesn't affect output at all, is used for side-effects only, and is ignored if it fails.

Reply:
That makes sense to me.